src/common/footer/Footer.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import Koros from 'hds-react/lib/components/koros/Koros';
+import { Link } from 'react-router-dom';
+import { useTranslation } from 'react-i18next';
+
+import tosConstants from '../../pages/memberhsip/constants/tosConstants';
+import { ReactComponent as HelsinkiLogo } from '../svg/HelsinkiLogo.svg';
+import styles from './Footer.module.css';
+
+function Footer() {
+  const { t, i18n } = useTranslation();
+  const selectedLanguage =
```

Comment:
I think you should use `i18n.languages[0]` to get the current language code ([source](https://www.i18next.com/overview/api#language))

Reply:
Yep, you are right. Changed.